From: Werner Fink <werner@ssue.de>
Date: Thu, 18 Oct 2018 05:32:42 +0000
Subject: Let cupsd start after possible network connection

For remote printer and printer servers, let cupsd always
start after a possible network connection.  This let cupsd
also stop before a used network connection goes down, hence
the cups does not lock due waiting on remote printers.

---
 scheduler/org.cups.cupsd.service.in |    1 +
 1 file changed, 1 insertion(+)

--- scheduler/org.cups.cupsd.service.in
+++ scheduler/org.cups.cupsd.service.in	2018-10-18 05:16:30.867333704 +0000
@@ -1,7 +1,7 @@
 [Unit]
 Description=CUPS Scheduler
 Documentation=man:cupsd(8)
-After=sssd.service
+After=sssd.service network.target
 
 [Service]
 ExecStart=@sbindir@/cupsd -l
